pkg/providers/aws/provider_postgres.go

```diff
@@ -671,7 +671,7 @@ func (p *PostgresProvider) getRDSConfig(ctx context.Context, r *v1alpha1.Postgre
 
 func (p *PostgresProvider) isLastResource(ctx context.Context, namespace string) (bool, error) {
 	listOptions := client.ListOptions{
-		Namespace: namespace,
+		Namespace: "",
```

Comment:
I think we can remove the namespace variable if it's unused elsewhere :thinking:

Reply:
:+1: